README.md

```diff
@@ -1 +1,181 @@
-# blog-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Blog app</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Blog app <a name="about-project"></a>
+
+**[Blog app]** is a functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Linters</summary>
+  <ul>
+    <li>Rubocop</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Use of Ruby on rails]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO 
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://reactcapstonproject.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+You need ruby and bundler
+
+### Setup
+
+Clone this repository to your desired folder:
+
+sh <br>
+cd my-folder <br>
+git clone https://github.com/Dachrono/blog-app.git
+
+### Install
+
+Install this project with:
+
+gem install rails
+
+### Usage
+
+To run the project, execute the following command:
+
+rails server
+
+### Run tests
+
+To run tests, run the following command:
+
+rspect
```

Comment:
### Optional
- [ ] It would be nice to fix the typo and make the testing section more detailed like `rspec spec`, this means `rspec` will run all the tests in the `spec` directory

Reply:
Tky I had not noticed that. Solved